### DISCUSSION TITLE
Disabling monitor crashes hyprland?

### DISCUSSION THREAD

**Discussion by hdev1** [hyprwm/Hyprland#926] [Q&A] [answered]
Hey,

My issue is that I want to disable my laptop's monitor (eDP-1) if my external monitor is plugged in (HDMI-A-1). I know I could do this with an external shell script with `exec-once` theoretically, but using `wlr-randr --output eDP-1 --off` seems to crash hyprland. 

I'm new to Wayland, just wondering if there's an obvious solution to this, or if hyprland crashing is upon disabling a montior with wlr-randr specific to my machine for some reason.

Thanks!

**Comment by vaxerski** [comment, marked as answer]
prefer using the config instead of `wlr-randr` or any other external utility. `hyprctl keyword monitor eDP-1,disable` should work. Remember to never disable the last monitor on your computer, it will freeze Hyprland.

**Reply by hdev1** [reply]
Thank you, that should do :)
